content/en/news/releases/1.10.x/announcing-1.10/change-notes/index.md

```diff
@@ -0,0 +1,179 @@
+---
+title: Change Notes
+linktitle: 1.10
+description: Istio 1.10.0 release notes.
+publishdate: 2021-05-18
+release: 1.10
+weight: 10
+aliases:
+    - /news/announcing-1.10
+---
+
+## Deprecation Notices
+
+These notices describe functionality that will be removed in a future release according to [Istio's deprecation policy](/docs/releases/feature-stages/#feature-phase-definitions). Please consider upgrading your environment to remove the deprecated functionality.
+
+- **Deprecated** the `values.global.jwtPolicy=first-party-jwt` option. This option is less secure and intended for backwards compatibility
+with older Kubernetes clusters without support for more secure token authentication but is now enabled by default in new Kubernetes versions.
+The option will be removed in future releases. See [this documentation](/docs/ops/best-practices/security/#configure-third-party-service-account-tokens) for more information.
+
+- **Deprecated** the `values.global.arch` option in favor of the affinity Kubernetes settings.
+  ([Issue #30027](https://github.com/istio/istio/issues/30027))
+
+## Traffic Management
+
+- **Added** `meshConfig.discoverySelectors` to dynamically restrict the set of namespaces for `Services`, `Pods`, and `Endpoints` that istiod processes when pushing xDS updates to improve performance on the data plane.
+  ([Blog](/blog/2021/discovery-selectors/), [Issue #26679](https://github.com/istio/istio/issues/26679))
+
+- **Added** the `ISTIO_GATEWAY_STRIP_HOST_PORT` environment variable to control whether gateways strip host port before any processing of request by HTTP filters or routing. This option is disabled by default.
```

Comment:
whether gateways strip **the** host port before any processing of request**s** by